plugins/i3s/io/EsriReader.hpp

```diff
@@ -109,7 +109,7 @@ class PDAL_DLL EsriReader : public Reader
       Bounds bounds;
       uint16_t threads = 8;
       std::vector<std::string> dimensions;
-      uint64_t lod;
+      double lod;
```

Comment:
Add test to SLPK tests of `lod` option

Reply:
I wrote one for i3s, but the slpk file we currently have only has one node and wouldn't show any valid pruning based on the lod.